code/controllers/subsystem/persistent_paintings.dm

```diff
@@ -184,6 +216,31 @@ SUBSYSTEM_DEF(persistent_paintings)
 			fdel(json_file)
 			WRITE_FILE(json_file, json_encode(result))
 			return update_format(result)
+		if(1) //Makes sure old paintings get a cosmetic frame type from their patronage tiers.
```

Comment:
This won't work for persistent paintings of version 0. They'll only be migrated up to 1, then not migrate again until next time. I know *we* don't have any, but this is bad practice for migrations.

Split these migrations into procs, like such:

```dm
if (version < 1)
    migrate_to_version_1()
if (version < 2)
    migrate_to_version_2()
```

...etc.